cognite/gqlpygen/main.py

```diff
@@ -1,60 +1,165 @@
 import importlib
 import importlib.util
 import inspect
+import os
+import subprocess
 import sys
 from pathlib import Path
 
 import click
 import typer
 
+from cognite.dm_clients.config import settings
 from cognite.dm_clients.domain_modeling.schema import Schema
 from cognite.gqlpygen.generator import to_client_sdk
 from cognite.gqlpygen.misc import to_client_name, to_schema_name
 
 app = typer.Typer()
 
 
-@app.command("topython", help="Input a .graphql schema to create pydantic schema.")
+_settings_cognite = settings.get("cognite", {})
+_cdf_cluster = _settings_cognite.get("cdf_cluster")
+_tenant_id = _settings_cognite.get("tenant_id")
+_client_id = _settings_cognite.get("client_id")
+_client_secret = _settings_cognite.get("client_secret")
+_project = _settings_cognite.get("project")
+
+_settings_dm_clients = settings.get("dm_clients", {})
+_space = _settings_dm_clients.get("space")
+_datamodel = _settings_dm_clients.get("datamodel")
+_schema_version = _settings_dm_clients.get("schema_version")
+
+_settings_local = settings.get("local", {})
+_prefix = _settings_local.get("prefix", "")
+_graphql_schema = _settings_local.get("graphql_schema")
+_schema_module = _settings_local.get("schema_module")
+
+_schema_dir = Path(_graphql_schema).parent if _graphql_schema else None
```

Comment:
I think all this unpacking is unnecessary if you use `pydantic` instead. The you can for example get the space as `settings.dm_clients.space`